bookkeeper-server/src/main/java/org/apache/bookkeeper/client/impl/LedgerEntriesImpl.java

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Can you add a test case `LedgerEntriesImplTest`? Like what I did for `LedgerEntryImplTest`?

Reply:
Thanks. will add it.